src/main/java/org/fogbowcloud/arrebol/ArrebolController.java

```diff
@@ -158,4 +192,41 @@ public TaskState getTaskState(String taskId) {
         //FIXME:
         return null;
     }
+
+    private void setWorkerCreator(String type) throws IOException {
```

Comment:
didn't like the name of this method. how about "initWorkerCreator"  or "loadWorkerCreator"? set methods have a very specific meaning in java

Reply:
Ok. I renamed it for loadWorkerCreator.